<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pb_0">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>460</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leINPUT">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>441</width>
      <height>61</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>370</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_8">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>190</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_9">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPandM">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPoint">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCLEAR">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPLUS">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCOMP">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDELT">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>100</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbMINUS">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbEQUAL">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>370</y>
      <width>80</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCLEARNUMBER">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>CE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbMEMORYSAVE">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>280</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>MS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbMEMORYPLUS">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>370</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>M+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbMEMORYMINUS">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>460</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>M-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbMEMORYRECALL">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>MR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbMEMORYCLEAR">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>80</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>MC</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
